bundles/org.openhab.binding.androiddebugbridge/src/main/java/org/openhab/binding/androiddebugbridge/internal/AndroidDebugBridgeDevice.java

```diff
@@ -202,6 +202,10 @@ public void stopPackage(String packageName)
     public String getCurrentPackage() throws AndroidDebugBridgeDeviceException, InterruptedException,
             AndroidDebugBridgeDeviceReadException, TimeoutException, ExecutionException {
         var out = runAdbShell("dumpsys", "window", "windows", "|", "grep", "mFocusedApp");
+        if (getAndroidVersion().contains("11")) {
```

Comment:
Can you provide an example how this string looks? Is is possible to get exact major version instead of having to rely on string comparison?

Reply:
Also, if this fix only for Android 11, or 11 and higher? The previous PR mentioned "before Android 11".

Reply:
I believe it is Android 11 or higher....  the string looks like this (this is from my pixel 6 pro hence the 12)
raven:/ $ getprop ro.build.version.release
12

there are other build props we could use.. that was just already there.

Reply:
what do you mean? that build prop has been around since android 1

Reply:
Sorry for being unclear: I meant implementation of getting version number might change because of other comment by @GiviMAD. Maybe it won't be string, in which case you won't have to parse it to get integer value for comparison.